index.html

```diff
@@ -36,7 +36,7 @@
     </ul>
     <h1>FAQ</h1>
     <h3>How did you make the video so cinematic?</h3>
-    <p>We're happy you asked, we followed a tutorial found on youtube and was surprised it worked.</p>
+    <p>We're happy you asked! We followed a tutorial on youtube and, well, here we are.</p>
```

Comment:
Classic, the pizzazz this adds is fabulous! :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: